packages/framework-provider-azure-infrastructure/src/infrastructure/index.ts

```diff
@@ -44,10 +45,14 @@ async function deployApp(logger: Logger, config: BoosterConfig, rockets?: Infras
 /**
  * Nuke all the resources used in the Resource Group
  */
-async function nukeApp(_logger: Logger, config: BoosterConfig): Promise<void> {
+async function nukeApp(_logger: Logger, config: BoosterConfig, rockets?: InfrastructureRocket[]): Promise<void> {
   const credentials = await azureCredentials()
   const resourceManagementClient = await createResourceManagementClient(credentials)
 
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```

Comment:
Why this `ts-ignore` was needed?

Reply:
TS complaints because the unmountStack could be undefined but that's not true as we are filtering the rockets where unmountStack are defined